woocommerce/woocommerce-android
# Lock shipping lines

Closes: #12869 

### Description
This PR fixes the issue of the shipping lines being editable at all times. It does that by introducing the `ShippingLineSection` class that includes the isEnabled state. Relying on the isEnabled state value, the shipping lines section now locks(disabled) the interaction with the shipping lines controls if the isEnabled property is false,

### Testing information

1. Go to Order List
2. Click on Create Order
3. Add a product
4. Notice Add shipping button is disabled when the order is refreshed
5. Add a shipping line
6. Notice the added shipping line is disabled when the order is refreshed
7. Add a product
8. Notice Add shipping button is disabled when the order is refreshed

### The tests that have been performed
1. Testing that the shipping lines section is disabled when the order is refreshing or is not editable.

### Images/gif

https://github.com/user-attachments/assets/5c87bad7-0993-44e5-a9cf-4539bd198466


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
